### ISSUE TITLE
Crash on nadir - Load Plan

### ISSUE THREAD

**Issue by ocacontrolroom** [astromg/TOI#33]
When loading plan, TOI crashes on the time calculation:

```python
observer@nadir:~/src/TOI$ poetry run toi
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
INFO:ob_config:Using config: /home/observer/src/ocabox/ob/config.yaml
INFO:ob_config:Empty config: /etc/ocabox/config.yaml
INFO:ob_config:Empty config: /etc/ocabox/.secrets.yaml
INFO:ob_config:Empty config: /home/observer/src/ocabox/ob/configuration/config.yaml
INFO:ob_config:Empty config: /home/observer/src/ocabox/ob/configuration/.secrets.yaml
INFO:client:Connecting to TIC server at tcp://tic.oca.lan:5558
INFO:base_async_widget:Starting task: 
Traceback (most recent call last):
  File "/home/observer/src/TOI/plan_gui.py", line 533, in loadPlan
    self.update_table()          
  File "/home/observer/src/TOI/plan_gui.py", line 122, in update_table
    self.update_plan()
  File "/home/observer/src/TOI/plan_gui.py", line 116, in update_plan
    slotTime = slotTime + (float(x_seq.split("/")[0]) * (float(x_seq.split("/")[2]) + float(self.parent.overhed)))
AttributeError: 'TOI' object has no attribute 'overhed'
Aborted (core dumped)
```

**Comment by ocacontrolroom** [comment]
Temporary fixed by commit 86a2551

**Comment by astromg** [comment]
it's already fixed in the branch developement_TOI, but not yet merged to main branch.
